article/equal-data.tex

```diff
@@ -1,17 +1,26 @@
 \subsection{Dataset} \label{equal-data}
-When measuring the performance of the different methods it is essential that the datasets are the same.
-Without them being equal the results of the experiments will not be worth comparing.
-Both PUP and LightGCN utilized the yelp dataset according to their papers.
-We did not get the dataset from the PUP implementation and were unable to find the original yelp-2018 dataset from the LightGCN implementation where the price and category were still attached.
-We therefore decided to utilize the yelp-2020 dataset, in which users rank businesses between 1 to 5 stars.
+Both PUP and LightGCN utilized the yelp-2018 dataset to evaluate their methods.
+We did not recieve a dataset from the PUP implementation and the LightGCN implementation's dataset did not preserve categories and pricew for items so could not be used to evaluate PUP.
```

Comment:
```suggestion
We did not receive a dataset from the PUP implementation and the LightGCN implementation's dataset did not preserve categories and prices for items so this could not be used to evaluate PUP.
```